src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -69,9 +69,44 @@ public void keyboardShortcutsTest() {
         assertEquals(UIComponentFocusEvent.EventType.FILTER_BOX, uiComponentFocusEventType);
         clearUiComponentFocusEventType();
         push(KeyCode.ESCAPE);
-        press(KeyCode.CONTROL).press(KeyCode.DOWN).release(KeyCode.DOWN).release(KeyCode.CONTROL);
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT2).release(KeyCode.CONTROL).release(KeyCode.DIGIT2);
         sleep(1000);
-        assertEquals(10, selectedIssueId);
+        assertEquals(9, selectedIssueId);
+        clearSelectedIssueId();
+        push(KeyCode.ESCAPE);
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT3).release(KeyCode.CONTROL).release(KeyCode.DIGIT3);
+        sleep(1000);
+        assertEquals(8, selectedIssueId);
+        clearSelectedIssueId();
+        push(KeyCode.ESCAPE);
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT4).release(KeyCode.CONTROL).release(KeyCode.DIGIT4);
+        sleep(1000);
```

Comment:
Could you see if the sleeps can be removed, or replaced with `PlatformEx.waitOnFxThread`?